operator/internal/decommissioning/statefulset_decomissioner.go

```diff
@@ -307,12 +307,6 @@ func (s *StatefulSetDecomissioner) Decommission(ctx context.Context, set *appsv1
 
 	log := ctrl.LoggerFrom(ctx, "namespace", set.Namespace, "name", set.Name).WithName("StatefulSetDecommissioner.Decomission")
 
-	// if helm is not managing it, move on.
-	if managedBy, ok := set.Labels[k8sManagedByLabelKey]; managedBy != "Helm" || !ok {
```

Comment:
👍

Reply:
is this safe for any use within the sidecar etc?

Reply:
It should be, I can add the label check to the sidecar's filter like you did in the `run.go` for the component label check.